docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -16,6 +16,18 @@ To connect to a ClickHouse server, you need:
 * Network access from the Trino coordinator and workers to the ClickHouse
   server. Port 8123 is the default port.
 
+.. note::
```

Comment:
Move to "Configuration" section.

Reply:
I want to add it after [this line](url), do you think it's OK?

Reply:
Could you fix the link? It reruns 404 not found.

Reply:
That looks okay.